### ISSUE TITLE
error whit version

### ISSUE THREAD

**Issue by Propants05** [plan-player-analytics/Plan#2338]
### Describe the issue
the plugin don't work wiht 1.14.5 paper

### Exceptions & Other Logs
[23:46:43] [Server thread/INFO]: [Plan] Enabling Plan v5.4 build 1633
[23:46:43] [Server thread/ERROR]: Error occurred while enabling Plan v5.4 build 1633 (Is it up to date?)
java.lang.NoSuchFieldError: a
	at javassist.f.<init>(Unknown Source) ~[?:?]
	at javassist.f.a(Unknown Source) ~[?:?]
	at com.djrapitops.plan.PlanL10.a(Unknown Source) ~[?:?]
	at com.djrapitops.plan.Plan.onEnable(Plan.java) ~[?:?]
	at org.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:263) ~[patched_1.14.4.jar:git-Paper-245]
	at org.bukkit.plugin.java.JavaPluginLoader.enablePlugin(JavaPluginLoader.java:338) ~[patched_1.14.4.jar:git-Paper-245]
	at org.bukkit.plugin.SimplePluginManager.enablePlugin(SimplePluginManager.java:420) ~[patched_1.14.4.jar:git-Paper-245]
	at org.bukkit.craftbukkit.v1_14_R1.CraftServer.enablePlugin(CraftServer.java:467) ~[patched_1.14.4.jar:git-Paper-245]
	at org.bukkit.craftbukkit.v1_14_R1.CraftServer.enablePlugins(CraftServer.java:381) ~[patched_1.14.4.jar:git-Paper-245]
	at net.minecraft.server.v1_14_R1.MinecraftServer.a(MinecraftServer.java:474) ~[patched_1.14.4.jar:git-Paper-245]
	at net.minecraft.server.v1_14_R1.DedicatedServer.init(DedicatedServer.java:290) ~[patched_1.14.4.jar:git-Paper-245]
	at net.minecraft.server.v1_14_R1.MinecraftServer.run(MinecraftServer.java:876) ~[patched_1.14.4.jar:git-Paper-245]
	at java.lang.Thread.run(Thread.java:829) [?:?]
[23:46:43] [Server thread/INFO]: [Plan] Disabling Plan v5.4 build 1633
[23:46:43] [Server thread/INFO]: [Plan] Player Analytics Disabled.

### Plugin versions
5.4 build 1633

**Comment by Kopo942** [comment]
TL;DR: You've installed some plugin from an unknown source that contains malware, and now your server has a RAT on it allowing the attacker to run whatever commands they please. It spreads to every plugin you install on the server. The `L10` in the stacktrace indicates the malicious plugin has injected itself into your Plan JAR, it makes its own class file with `L10` at the end in the name.

The only fix for this is to delete **everything** except your world folder(s) and completely reinstall your server.

**Comment by Propants05** [comment]
oh...... okay

**Comment by AuroraLS3** [comment]
Yeah looks like this isn't caused by Plan code since there is no PlanL10 class in the jar by default and obfuscation is not used by Plan. I think Kopo knows what he's talking about so I'll close this.
